### PR TITLE
Update shared prototypes

### PR DESCRIPTION
**Pull request by Alexander-Degtyar**

### CODE REVIEW

**Review by ValentiWorkLearning — APPROVED** [review]
Ship it 😁

**Comment by Alexander-Degtyar** [comment]
> Ship it 😁

Thank you very much @ValentiWorkLearning. We appreciate it.

File: src/shared/protos/actions.proto [hunk 1]
```diff
@@ -1,87 +1,98 @@
...
-  }
-  opengreenery.common.VentilationMode mode = 1;
-  oneof configuration {
-    Auto auto = 2;
-    Manual manual = 3;
+    google.protobuf.Duration enabling_time = 1;
+    google.protobuf.Duration disabling_time = 2;
   }
+  open_greenery.common.VentilationMode mode = 1;
+  open_greenery.common.VentilationState = 2;
```
Comment (seclerp) [inline src/shared/protos/actions.proto]: Field name is missing

File: src/shared/protos/actions.proto [hunk 2]
```diff
@@ -1,87 +1,98 @@
...
+    MoistureLevel dry_level = 1;
+    MoistureLevel wet_level = 2;
     google.protobuf.Duration watering_period = 3;
-    double watering_volume = 4;
+    uint32 watering_volume_ml = 4;
   }
-  opengreenery.common.SoilIrrigationMode mode = 1;
+
+  AreaId area_id = 1;
+  open_greenery.common.SoilIrrigationMode mode = 1;
```
Comment (seclerp) [inline src/shared/protos/actions.proto]: Adjust correct field numbers, please

**Review by seclerp — CHANGES_REQUESTED** [review]

**Comment by Alexander-Degtyar** [comment]
@seclerp, fixed.

**Review by seclerp — APPROVED** [review]

**Pull request merged**